org.eclipse.cdt.cross.arc.gnu/src/org/eclipse/cdt/cross/arc/gnu/common/StateConstants.java

```diff
@@ -0,0 +1,25 @@
+/*******************************************************************************
+ * This program and the accompanying materials are made available under the terms of the Common
+ * Public License v1.0 which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/cpl-v10.html
+ *
+ * Copyright (c) 2016 Synopsys, Inc.
+ *******************************************************************************/
+
+package org.eclipse.cdt.cross.arc.gnu.common;
+
+/**
+ * This class contains all constants for accessing and checking a compiler plug-in' state.
+ */
+public class StateConstants {
```

Comment:
As discussed offline, you should merge this class with `StateChecker` and `StateWriter` (or whatever it new name will be) shold get names and values as arguments to its functions.